### DISCUSSION TITLE
Cross validation

### DISCUSSION THREAD

**Discussion by mssanamateen** [mssanamateen/MLAlgorithms#20] [Q&A] [answered]
What is cross validation? Share your thoughts

**Comment by Mazin7d4** [comment, marked as answer]
Cross validation is a technique which is used to check how well a model performs. It involves dividing the data into parts, training the model on some parts and testing it on others. This helps ensure that the model works well with new, unseen data.

**Comment by Haarika78** [comment]
Cross-validation is a technique for evaluating a machine learning model and testing its performance. CV is commonly used in applied ML tasks. It helps to compare and select an appropriate model for the specific predictive modeling problem.
